e2e/backend.e2e.spec.ts

```diff
@@ -143,10 +140,7 @@ test.describe.serial('KubernetesBackend', () => {
       };
 
       /** Platform-specific additions to `expectedDefinition`. */
-      const platformExpectedDefinitions: Partial<Record<NodeJS.Platform, ExpectedDefinition>> = {
-        win32:  { 'virtualMachine.hostResolver': false },
-        darwin: { 'experimental.virtualMachine.socketVMNet': false },
-      };
+      const platformExpectedDefinitions: Partial<Record<NodeJS.Platform, ExpectedDefinition>> = { win32: { 'virtualMachine.hostResolver': false } };
```

Comment:
See above — we should still have a platform-specific thing (to test that things switch off correctly on Windows).